core/chains/evm/config/oracletype/oracletype.go

```diff
@@ -0,0 +1,17 @@
+package oracletype
+
+type OracleType string
```

Comment:
Is it verbose to call the package `daoracletype` and the type `DAOracleType` ? It's a little more clear to me what the scope of it should be with that naming

Reply:
Hmmm the linter should be flagging this. There is a linter rule that the type should not repeat the package name, based on effective go naming conventions:
> The importer of a package will use the name to refer to its contents, so exported names in the package can use that fact to avoid repetition. (Don't use the import . notation, which can simplify tests that must run outside the package they are testing, but should otherwise be avoided.) For instance, the buffered reader type in the bufio package is called Reader, not BufReader, because users see it as bufio.Reader, which is a clear, concise name. Moreover, because imported entities are always addressed with their package name, bufio.Reader does not conflict with io.Reader. Similarly, the function to make new instances of ring.Ring—which is the definition of a constructor in Go—would normally be called NewRing, but since Ring is the only type exported by the package, and since the package is called ring, it's called just New, which clients of the package see as ring.New. Use the package structure to help you choose good names.

https://go.dev/doc/effective_go#names